README.md

```diff
@@ -30,6 +32,8 @@ pip install ansible vcert --upgrade
 - [`venafi.machine_identity.certificate`](roles/certificate/README.md): Enrolls a certificate and optionally deploys it to a remote location.
 - [`venafi.machine_identity.policy`](roles/policy/README.md): Creates or updates certificate policy on VaaS or TPP using a specification file.
 
+<!-- DW: So this readme is not parallell to the other readmes, which breaks the content model and makes it more difficult for customers to get used to know where and how to find information. For example, where's the "Test driver our examples" section, or the Setup section? How do i know what to do next to get started? What steps do I need to take following meeting the requirements? -->
+
```

Comment:
I think the content you were expecting is part of the [certificate role README](https://github.com/Venafi/ansible-collection-venafi/blob/iis-example/roles/certificate/README.md), right?

Reply:
Yes, the goal being to make our content as consistent and repeatable as possible for the sake of our customers. So where it makes sense, that is for similar doc deliverables, that we use the same flow. Customers then learn the flow and come back more often because they're familiar with it (assuming it's good to start with and they have a positive experience). I suggest doing that here, but if it's important to get this out quickly, you could push it as-is and then make those incremental enhancements

Reply:
Agreed, I think what you are recommending actually needs to be done to the Terraform project because it currently has TLS certificate issuance and policy management together on the main README and it needs to be updated to speak to SSH certificates.  That is, there should be a separate README focused on each of those use cases rather than one big run-on.